brainstormforce/ultimate-addons-for-gutenberg
# wip: improvement: Typography CSS generation logic.

### Description
This PR improves the Typography CSS generation logic.

### Types of changes
Logical improvement. Could break frontend CSS, if something is missed.

### How has this been tested?
- Create pages with all blocks.
- Checkout to this branch.
- One must see no change in the frontend.

### Checklist:
